src/js/game_logic.js

```diff
@@ -39,6 +34,8 @@ class RhymusGame {
     constructor() {
         this.currentCard = undefined
         timerDisplay.textContent = getTimerString(timeRemaining)
+        rhymeElement.textContent = 'Press start to Play!'
```

Comment:
Capitalization is a little weird here. Let's make "Play" lowercase as well for consistency.